dask/array/core.py

```diff
@@ -4553,4 +4549,34 @@ def from_npy_stack(dirname, mmap_mode="r"):
     return Array(dsk, name, chunks, dtype)
 
 
+_HANDLED_FUNCTIONS = {}
+
+
+def implements(numpy_function):
```

Comment:
You might just write `*numpy_functions` here, then you could skip the introspection below.

Reply:
done, thanks for the suggestions